library/psa_crypto.c

```diff
@@ -5474,6 +5474,123 @@ static psa_status_t psa_key_derivation_tls12_ecjpake_to_pms_read(
 }
 #endif
 
```

Comment:
Missing: a patch to `psa_key_derivation_setup_kdf`. This is partly preexisting (it's also wrong for TLS12_PRF) so let's do it [in a follow-up](https://github.com/Mbed-TLS/mbedtls/issues/7711).